src/main/java/Duke/Storage.java

```diff
@@ -0,0 +1,132 @@
+package Duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Storage {
+    /**
+     * Saves the list of Tasks to hard drive
+     * Will be called whenever a task is added or deleted
+     *
+     * @param userLists ArrayList of user tasks
+     */
+    public static void saveList(ArrayList<Task> userLists) {
+        String pathName = "./data/duke.txt";
```

Comment:
You could declare this file path as a [constant](https://se-education.org/guides/conventions/java/basic.html#naming) if it is not modified anywhere else.